tests/framework/utils/helm_helper.go

```diff
@@ -0,0 +1,87 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/pkg/util/exec"
+)
+
+var hhlogger = capnslog.NewPackageLogger("github.com/rook/rook", "helmhelper")
```

Comment:
i'd recommend one logger per package. This will also encourage a pattern of creating more specific packages instead of creating a `util` package that is general.